backend/services/news/src/schema.graphql

```diff
@@ -21,6 +21,8 @@ type Article @key(fields: "id") {
   publishedDatetime: Datetime!
   imageUrl: Url
   latestEditDatetime: Datetime
+  likes: Int!
+  isLiked: Boolean!
```

Comment:
I find this name sort of confusing, maybe name it "isLikedByMe" since we use "me" in other queries

Reply:
agreed